test/api/assignments_test.rb

```diff
@@ -29,7 +29,8 @@ def test_api_complains_if_no_key_is_submitted
   def test_api_accepts_submission_attempt
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
-        post '/user/assignments', {key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb'}.to_json
+        post '/user/assignments', {key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 'language' => 'ruby',
```

Comment:
Line is too long. [116/80]<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.